async-seq.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const path = require('path');
+const async = require('async');
+const R = require('ramda');
+const S = require('sanctuary');
+
+const join = (x, cb) => cb(null, path.join(process.argv[2], x));
+const map = R.flip(async.map);
+
+const output = (err, results) => {
+  if (err) {
```

Comment:
That's the case: async returns ```undefined```. I don't know why and maybe this is currently error.